lib/registries/addon/components/registries-navbar/template.hbs

```diff
@@ -101,43 +102,53 @@
                     data-test-login='1'
                     data-analytics-name='Sign in'
                     @onclick={{action 'login'}}
+                    @classNames={{local-class 'HideOnMobile'}}
                 >
                     <h4>{{t 'navbar.login'}}</h4>
                 </nav.buttons.secondary>
+            {{/unless}}
 
-            {{else}}
-
-                <BsDropdown
-                    data-analytics-scope='User dropdown'
-                    local-class='AuthDropdown'
-                    as |dd|
+            <BsDropdown
+                data-analytics-scope='User dropdown'
+                local-class='AuthDropdown'
+                as |dd|
```

Comment:
Please use a more descriptive name than "dd", it's hard to tell what context this is apart of.